CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Contributions Welcome!
+
+This repository is part of the Fabric project.
+Please consult [Fabric's CONTRIBUTING documentation](http://hyperledger-fabric.readthedocs.io/en/latest/CONTRIBUTING.html) for information on how to contribute to this repository. 
```

Comment:
This should hopefully be accurate without any caveats eventually.

For this text, it might make some sense to mention that we're doing the github work style.